.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
```

Comment:
It's in conflict with windows default, which is the point.  I'm not sure I see the advantage of adding .gitattributes when this line of the .editorconfig should do the trick.  The final result is `lf` in the repo.